pkgs/top-level/all-packages.nix

```diff
@@ -8744,7 +8744,7 @@ in
 
   fish = callPackage ../shells/fish { };
 
-  fish-foreign-env = callPackage ../shells/fish/fish-foreign-env { };
+  fishPlugins = recurseIntoAttrs (callPackage ../misc/fish-plugins { });
```

Comment:
I think this should go into a new `shells/fish/plugins` directory rather than `misc/fish-plugins`.

Reply:
Indeed. I moved the directory as suggested and amended.